CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished Changes
+
+
+## 1.37.2 10-27-2022
 * larva-patterns - Remove extra closing div from `faq` module.
+* * larva-tokens - Update larva tokens for spy-2022.
```

Comment:
Can we remove this extra bullet on this line. Else the PR looks good.

Reply:
Updated.